Shared/configurations/ConfigurationListModel.cpp

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+ *  Copyright 2012-2025 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+#include "pch.hpp"
+
+#include "ConfigurationListModel.h"
+
+namespace Dsa {
+
+ConfigurationListModel::ConfigurationListModel(QObject* parent):
+  QAbstractListModel(parent)
+{
+}
+
+ConfigurationListModel::~ConfigurationListModel()
+{
+}
```

Comment:
```suggestion
ConfigurationListModel::~ConfigurationListModel() = default;
```

Throughout for any empty constructors or destructors.

Reply:
removed per our discussion in teams
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#c30-define-a-destructor-if-a-class-needs-an-explicit-action-at-object-destruction